### DISCUSSION TITLE
Map HL7 format message to XML

### DISCUSSION THREAD

**Discussion by ghost** [atlasmap/atlasmap#3427] [Q&A]
Hello all,

Can we use Atlasmap to map HL7 format message to XML? 

I am not able to import HAPI library, which has the Java files for HL7 message format, in Atlasmap UI

Please suggest the right way to import the HL7 format definitions into Atlasmap to enable the mapping into XML

Many thanks

**Comment by igarashitm** [comment]
What happens when you import that Java library? Did you add Java class as a Document?
https://docs.atlasmap.io/#importing-java-archive-files-into-atlasmap

**Comment by ghost** [comment]
After I imported all the java archive files needed, I can see the class list. Thanks!
